test/aggregations-test/composite-agg-values-sources-test/terms-values-source.test.js

```diff
@@ -36,6 +36,7 @@ test('constructor sets arguments', t => {
 test(setsAggType, TermsValuesSource, 'terms');
 test(validatedCorrectly, getInstance, 'order', ['asc', 'desc']);
 test(setsOption, 'missing', { param: 42 });
+test(setsOption, 'missingBucket', { param: 42 });
```

Comment:
Sorry, I am new to AVA test style, so not sure this test is fine.
I just copy-n-pasted this from a test for `missing`
I interpreted these tests as `Set option "missing" with parameter "42"` which is weird for me, because `missing` should accept `string`.
I will fix test on your guidance.

Reply:
It is less to do with `ava` and more to do with the way I wrote the tests. I went overboard with deduplicating code and added too many helpers. What you did was close enough.

`missing` allows setting an explicit value when the value is `null` so a `42` is valid. However, the jsdoc and type definitions weren't accurate so I have pushed a commit which fixes it.